IntelFsp2Pkg/Include/FspEas/FspApi.h

```diff
@@ -11,19 +11,24 @@
 #define _FSP_API_H_
 
 #include <Pi/PiStatusCode.h>
+#include <Base.h>
 
 ///
 /// FSP Reset Status code
-/// These are defined in FSP EAS v2.0 section 11.2.2 - OEM Status Code
+/// These are defined in FSP EAS v2.4 section 13.3 - OEM Status Code
```

Comment:
@bejeanmo small correction, section # is 13.2.2 if there is not change in EAS ver2.4 Errata A .